haiku-apps/beohms/beohms-0.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="Ohm's Law calculator"
+DESCRIPTION="BeOhms is a calculator that is based of of Ohm's Law. This \
+calculator solves either voltage, resistance, or current, given two of the \
+variables."
+HOMEPAGE="https://github.com/HaikuArchives/BeOhms/"
+COPYRIGHT="1999 Kyle Crane"
+LICENSE="Public Domain"
+REVISION="1"
+commit="43cf57c95ae70a98c64e01c4d600eaa0ca7519ed"
+SOURCE_URI="https://github.com/HaikuArchives/BeOhms/archive/$commit.tar.gz"
+CHECKSUM_SHA256="65901b77d0deb15b0b327a38b0965382d171ff178f3737fadd4a3741aa7c9b14"
+SOURCE_FILENAME="BeOhms-$commit.tar.gz"
+SOURCE_DIR="BeOhms-$commit"
+PATCHES="beohms-0.1.patchset"
+ADDITIONAL_FILES="Makefile"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	beohm$secondaryArchSuffix = $portVersion
+	app:BeOhm$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	makefile_engine
+	"
+
+BUILD()
+{
+	cp $portDir/additional-files/Makefile $sourceDir/src/Makefile
+	cd src
+	mkdir -p bin
+	make $jobArgs TARGET_DIR=bin
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+
+	cp src/bin/BeOhms $appsDir/BeOhms
```

Comment:
No need to repeat the name of the target, since you are not renaming it.
```
	cp src/bin/BeOhms $appsDir
```
or
```
	cp src/bin/BeOhms $appsDir/
```
if you frefer.

Wait, there is even a much better alternative:
```
	install -d $appsDir
	install -t $appsDir src/bin/BeOhms
```
(It will replace the mkdir and the cp.)